sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -164,6 +167,24 @@ def _closest_power_of_two(x: int) -> int:
     return 1 << ((x - 1).bit_length() - 1)
 
 
+def get_quota_code(accelerator: str, use_spot: bool) -> Optional[str]:
+    # Get the quota code from the accelerator instance type
+    # This will be used in the botocore command to check for
+    # a non-zero quota
```

Comment:
Can we change this to standard docstr format (starting with `"""`, like we have for other methods?)

Reply:
Same for get_quota_code in `aws_catalog.py`.. sorry we missed that earlier 😅